webapp/frontend/src/app/services/scan.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+import {Exam} from "../common/exam";
+import {Observable} from "rxjs";
+import {HttpClient} from "@angular/common/http";
+import {ScanData} from "../common/scan-data";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ScanService {
+  saveUrl = 'http://localhost:8080/api/scan/save';
```

Comment:
@MateuszKepczynski the same for frontend, addresses should be stored in one place for easy change